test/decl/protocol/special/coding/struct_codable_failure_diagnostics.swift

```diff
@@ -6,14 +6,6 @@ struct S1 : Codable {
   var a: String = ""
   var b: Int = 0
   var c: Nested = Nested()
-
-  // CHECK: error: type 'S1' does not conform to protocol 'Decodable'
```

Comment:
The ordering of these diagnostics has changed. Since we perform `Decodable` conformance checking in `addImplicitConstructors`, we hit all `Decodable` conformances first, and `Encodable` checking happens afterwards. These `CHECK`s can no longer be interleaved, so they're at the bottom of the file.